src/bundle.js

```diff
@@ -35,6 +34,5 @@ const requireComponents = (context) => {
 
 export const inputs = requireComponents(Inputs);
 export const layouts = requireComponents(Layouts);
-export const parser = requireComponents(Parser);
```

Comment:
The parsers should still be contained in the bundle so we should keep these lines.